xarray/conventions.py

```diff
@@ -659,7 +659,23 @@ def decode_cf(
 
     Returns
     -------
-    decoded : Dataset
+    Union[xr.Dataset, xr.DataArray]: The decoded xarray Dataset or DataArray
+
+
+    Example
+    -------
+    import xarray as xr
+
+    # Load an example NetCDF file
+    ds = xr.open_dataset('example.nc')
```

Comment:
The dataset is decoded by default when opening with `xr:open_dataset`. To make the example work as intended you would need to add kwarg `decode_cf=False`.